sway/commands/permit.c

```diff
@@ -54,6 +60,10 @@ static struct feature_policy *get_policy(const char *name) {
 		}
 		list_add(config->feature_policies, policy);
 	}
+	sway_assert(!strcmp(program, policy->program),
+			"!! DANGER !! symlink modified during policy allocation.");
```

Comment:
No, it logs the error and returns the first parameter. Sway should never abort at runtime, because that would kill your whole session and cause data loss and things like that. It's just a utility function to add conditional logging, the user has to implement a safe exit path.